.github/workflows/update-ci-ar-cleanup-policy.yml

```diff
@@ -0,0 +1,85 @@
+# Copyright 2023 The Authors (see AUTHORS file)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This workflow will be manully triggered to update CI project's
+# artifact registry clean up policy. This should be set up by terraform,
+# however it's not supported yet. We will remove this and use terraform
+# to set up this once it's supported.
+# To enable clean up policy, the ci service need to be granted the role
+# of [roles/artifactregistry.admin]
+
+# https://cloud.google.com/artifact-registry/docs/repositories/cleanup-policy
+
+name: 'update-ci-ar-cleanup-policy'
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - 'main'
```

Comment:
This pull_request condition will be removed before merging, it only for triggering dry run this workflow.